packages/coil-extension/src/background/services/BackgroundScript.ts

```diff
@@ -725,14 +725,28 @@ export class BackgroundScript {
 
     const receiver = stream.getPaymentPointer()
     const { assetCode, assetScale, exchangeRate } = stream.getAssetDetails()
+
+    // Set tip amount
     const tipAmount = tip < 1 ? 1 : tip
     const amount = Math.floor(tipAmount * 1e9 * exchangeRate).toString() // 1 USD, assetScale = 9
 
+    // Set tip origin url
+    async function tabQueryPromise() {
+      return new Promise(function (res, rej) {
+        chrome.tabs.query({ active: true, currentWindow: true }, res)
+      })
+    }
+    const activeTabUrl = await tabQueryPromise().then(
+      (tabs: any) => tabs[0].url
+    )
+
     try {
       this.log(`sendTip: sending tip to ${receiver}`)
-      const result = await this.client.initiateTip({
+
+      const result = await this.client.initiateTip(token, {
         amount,
-        destination: receiver
+        destination: receiver,
+        origin: activeTabUrl
```

Comment:
This doesn't really seem inline with the rest of our care with privacy

Reply:
So this is so we can show the domain in the list of tips right:
https://coiltech.slack.com/archives/C01MTJ37Y4D/p1617909873005700

Reply:
re: https://coiltech.slack.com/archives/C01MTJ37Y4D/p1617911934007000

Just a note that when I saw that was sending the full url as the `origin` it was a bit confusing.
When I read origin, tend to think of the base of the url:
`const actualOrigin = new URL(frame.href).origin`